<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>955</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="central_window">
   <widget class="QWidget" name="game_window" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>641</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="restart_button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>580</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>RESTART</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_label">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>60</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>箱子剩余移动次数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hint_Button">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>210</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>提示</string>
    </property>
   </widget>
   <widget class="QLabel" name="step_label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>100</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 75 64pt &quot;Songti SC&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>955</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
